extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -472,4 +475,34 @@ describe('amp-analytics', function() {
       expect(sendRequestSpy.args[0][0]).to.equal('https://example.com/remote');
     });
   });
+
+  describe('data-consent-notification-id', () => {
+
+    it('should resume fetch when consent is given', () => {
+      const analytics = getAnalyticsTag({
+        'requests': {'foo': 'https://example.com/local'},
+        'triggers': [{'on': 'visible', 'request': 'foo'}]
+      }, {
+        'data-consent-notification-id': 'amp-user-notification1'
+      });
+      analytics.consentPromise_ = Promise.resolve();
```

Comment:
Instead of this, do something like this:

```js
        const uidService = installUserNotificationManager(win);
        sandbox.stub(uidService, 'get', id => {
          expect(id).to.equal('amp-user-notification1');
          return  Promise.resolve();
        });
```

You might also want to check that that function gets called. You could put a spy on the stub :) Or just inc a var.